src/markdown/jobs/shared.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+export const About = () => (
+  <>
+    <p>
+      We believe that today's "digital identity" is fundamentally flawed. There
+      are too many profiles, registration screens, passwords, privacy
+      violations, data leaks, phishing attacks, and data mining operations out
+      there. Our vision is to make the internet more secure and private with
+      better software, open-source, and open knowledge.
+    </p>
+    <p>
+      Ory is the largest open-source authentication and authorization (login,
+      registration, MFA, permissions, OAuth2, ...) ecosystem and community in
+      the world. We develop open-source software and use it to build a scalable,
+      easy-to-use, and secure cloud platform. You will work on one of the
+      industry's most important code-bases, having a positive impact on hundreds
```

Comment:
```suggestion
      industry's most important code-bases, having a positive impact on hundreds
```
This sentence is problematic when we use a boilerplate `About` for all job openenings, since not everyone is working on codebase.